File: DSA/Arrays/carry-forward/max-min-of-array/python/minMaxOfArray.py
def solve(self, A):
        max = A[0];
        min = A[0];
        
        for i in range(1, len(A)):
            if max < A[i]:
                max = A[i] 
            elif min > A[i]:
                min = A[i]

        return max + min

File: DSA/Arrays/carry-forward/max-min-of-array/javascript/minMaxOfArray.js
export const minMaxOfArray = (A) => {
  let max = A[0];
  let min = A[0];
 
  for(let i = 1; i < A.length; i++){
      if (max < A[i]) {
          max = A[i] 
      } else if (min > A[i]){
          min = A[i]
      }
  }
  return parseInt(max + min)
}